pkg/master/master.go

```diff
@@ -35,6 +35,7 @@ import (
 	authorizationapiv1beta1 "k8s.io/api/authorization/v1beta1"
 	autoscalingapiv1 "k8s.io/api/autoscaling/v1"
 	autoscalingapiv2beta1 "k8s.io/api/autoscaling/v2beta1"
+	autoscalingapiv2beta2 "k8s.io/api/autoscaling/v2beta1"
```

Comment:
you miss-copied ;-)

It says `/autoscaling/v2beta1, you need `/autoscaling/v2beta2`

Reply:
Ah! Technically I copied it perfectly, it's the changing it that I messed up. Thanks for catching that